app/views/validators/_table.html.erb

```diff
@@ -100,7 +100,12 @@
         <% end %>
         <% if validator.score.authorized_withdrawer_score == -2 %>
           <%= link_to '/faq#withdraw-authority-warning', title: "Withdrawer matches validator identity." do %>
-            <i class="fas fa-exclamation-triangle text-warning ml-1"></i>
+            <i class="fas fa-exclamation-triangle text-danger ml-1"></i>
```

Comment:
Unwanted change, should be:
`<i class="fas fa-exclamation-triangle text-warning ml-1"></i>`